compiler/runtime/runtime.c

```diff
@@ -129,3 +129,26 @@ void* SKIP_context_sync(uint64_t txTime, char* old_root, char* delta,
   SKIP_call_after_unlock(synchronizer, delta);
   return new_root;
 }
+
+int64_t SKIP_Unsafe_Ptr__toInt(char* ptr) {
+  return (int64_t)ptr;
+}
+
+void* SKIP_Unsafe_array_ptr(char* arr, int64_t offset) {
```

Comment:
```suggestion
void* SKIP_Unsafe_array_ptr(char* arr, SkipInt offset) {
```
I guess this would document the fact that the value comes from skip.

(similarly at other places where it makes sense)